crates/spl_network_messages/src/lib.rs

```diff
@@ -18,17 +18,34 @@ pub use game_controller_state_message::{
     GameControllerStateMessage, GamePhase, GameState, Half, Penalty, PenaltyShoot, Player,
     SubState, Team, TeamColor, TeamState,
 };
-pub use visual_referee_message::{VisualRefereeDecision, VisualRefereeMessage};
+pub use visual_referee_message::{GestureVisualRefereeDecision, GestureVisualRefereeMessage};
+
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub enum HulkMessage {
+    Striker(StrikerMessage),
+    VisualReferee(VisualRefereeMessage),
+}
+
+impl Default for HulkMessage {
+    fn default() -> Self {
+        HulkMessage::Striker(StrikerMessage::default())
+    }
+}
 
 #[derive(Clone, Copy, Debug, Default, Deserialize, Serialize)]
-pub struct HulkMessage {
+pub struct StrikerMessage {
     pub player_number: PlayerNumber,
     pub pose: Pose2<Field>,
-    pub is_referee_ready_signal_detected: bool,
     pub ball_position: Option<BallPosition<Field>>,
     pub time_to_reach_kick_position: Option<Duration>,
 }
 
+#[derive(Clone, Copy, Debug, Default, Deserialize, Serialize)]
+pub struct VisualRefereeMessage {
+    pub player_number: PlayerNumber,
+    pub is_referee_ready_signal_detected: bool,
```

Comment:
Isn't the mere existence of the message enough? Do we need an additional bool?

Reply:
you are absolutely right, removed